README.adoc

```diff
@@ -574,6 +574,30 @@ class SomethingTest < Minitest::Test
 end
 ----
 
+=== Skipping [[skipping]]
```

Comment:
It would be good if the skip target is clarified. Although the term is difficult, "runnable method" has the following usage example.
https://github.com/minitest/minitest/blob/v5.17.0/lib/minitest/test.rb#L69

```suggestion
=== Skipping Runnable Method [[skipping-runnable-method]]
```